### ISSUE TITLE
Manuscript.Rmd with the PLOS template is not created 

### ISSUE THREAD

**Issue by nehamoopen** [cjvanlissa/worcs#123]
When initializing a new WORCS project with a PLOS manuscript template, a manuscript folder is created but there are no files within it. This error popped up for a participant at a WORCS workshop and I see that it doesn't work for me either.

My R session info is as follows:

```
R version 4.0.2 (2020-06-22)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Dutch_Netherlands.1252  LC_CTYPE=Dutch_Netherlands.1252   
[3] LC_MONETARY=Dutch_Netherlands.1252 LC_NUMERIC=C                      
[5] LC_TIME=Dutch_Netherlands.1252    

attached base packages:
[1] stats     graphics  grDevices datasets  utils     methods   base     

loaded via a namespace (and not attached):
 [1] compiler_4.0.2    htmltools_0.5.1.1 tools_4.0.2       yaml_2.2.1        rmarkdown_2.14   
 [6] knitr_1.39        xfun_0.31         digest_0.6.27     rlang_0.4.11      renv_0.13.2      
[11] evaluate_0.15 
```

![image](https://user-images.githubusercontent.com/37183829/175972589-2de5d843-6928-4992-852a-c4bdf5b56a5d.png)

![image](https://user-images.githubusercontent.com/37183829/175972963-f023947f-638f-44c6-979c-35962764902b.png)

**Comment by nehamoopen** [comment]
The issue remains when I use the `add_manuscript` function as follows:

```
> add_manuscript(".", "plos_article")
X  Creating manuscript files.
```

I also tried it with other article templates (as follows) and it turns out none of them worked. A manuscript folder is created, but there are no files within it.

```
> add_manuscript(".", "springer_article")
X  Creating manuscript files.

> add_manuscript(".", "jss_article")
X  Creating manuscript files.
```

Lastly, I tried the default option like so:

```
> add_manuscript(".")
X  Could not generate an APA6 manuscript file, because the 'papaja' package is not installed. Run this code to see instructions on how to install this package from GitHub:
  vignette("setup", package = "worcs")
v  Creating manuscript files.
```

I knew `papaja` was not loaded, I was just playing around. But this attempt created a manuscript folder that also contained a "references.bib" file, the previous attempts with PLOS/Springer/JSS did not generate even that.

**Comment by cjvanlissa** [comment]
Thank you @nehamoopen ! This should be fixed by #8e9002d
